src/tests/system/tests/test_ldap.py

```diff
@@ -502,3 +502,40 @@ def test_ldap__password_change_no_grace_logins_left(
 
     rc, _, _, _ = client.auth.parametrize(method).password_with_output("user1", "Secret123")
     assert rc == expected, err_msg
+
+
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_ldap__empty_attribute(client: Client, ldap: LDAP):
+    """
+    :title: SSSD fails to store users if any of the requested attribute is empty
+    :setup:
+        1. Disable Syntax Checking
+        2. Add a User
+        3. Make home attribute of user empty
+        4. Create an Organizational Unit
+        5. Add Groups
+        6. Start SSSD
+    :steps:
+        1. The user can be queried (id command)
+        2. Groups are resolved properly using the getent group command
```

Comment:
s/Groups are resolved properly using the getent group command/Groups are resolved

No need to explain how in the steps, just the what.